bin/saya/saya.sh

```
@@ -0,0 +1,184 @@
+fi
+
+if [[ -z "${SAYA_FORK_BLOCK_NUMBER}" ]]; then
+    echo "Set SAYA_FORK_BLOCK_NUMBER to the latest block including the preparations (check here https://sepolia.starkscan.co/, remember to switch to sepolia!)."
+    echo "You can now run \`cargo run -r --bin katana -- --rpc-url $SAYA_SEPOLIA_ENDPOINT --fork-block-number \$SAYA_FORK_BLOCK_NUMBER\` in another terminal."
+    exit 0
+fi
+
+if [[ -z "${SAYA_PILTOVER_ADDRESS}" ]]; then
+    sncast -a $SAYA_SNCAST_ACCOUNT_NAME deploy \
+        --class-hash $SAYA_PILTOVER_CLASS_HASH \
+        -c $SAYA_SEPOLIA_ACCOUNT_ADDRESS $SAYA_PILTOVER_STARTING_STATE_ROOT $(expr $SAYA_FORK_BLOCK_NUMBER + 1) 0 \
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)